subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/KotlinBuildScript.kt

```diff
@@ -56,7 +47,7 @@ annotation class KotlinScriptTemplate
 @GradleDsl
 abstract class KotlinBuildScript(
     private val host: KotlinScriptHost<Project>
-) : ProjectDelegate() {
+) : ProjectDelegate() /* TODO:kotlin-dsl configure Project as implicit receiver */ {
```

Comment:
For IDE support, right!

Reply:
This one is still the template given to the IDE for content assistance so it should match the others. This will be done as a follow-up issue: https://github.com/gradle/gradle/issues/10926